integration_test.go

```diff
@@ -261,9 +261,9 @@ func TestVerifyData(t *testing.T) {
 		aliases = append(aliases, db.image)
 		// Create db and connect
 		_, port, err := createContainer(t, ctx, db.image, db.port, db.env, db.cmd)
-		assert.NoError(t, err)
```

Comment:
What's the difference between the assert and require modules? I never really understood it.